tests/test_covariance_calculator.py

```diff
@@ -83,7 +83,11 @@ def test_create_sacc_cov():
     cc = CovarianceCalculator(input_yml)
     cov = cc.get_covariance() + 1e-100
 
-    cc.create_sacc_cov()
+    # Check returned file
+    s = cc.create_sacc_cov()
+    cov2 = s.covariance.covmat + 1e-100
+    assert np.max(np.abs(cov / cov2 - 1) < 1e-10)
```

Comment:
There's also a pytest way of doing these almost equals:

```
assert test == pytest.approx(ref, 1e-10)
```
Definitely not required change though

Reply:
I'll leave it like that for now but looks pretty convenient. Thanks!